integration/jaxrs/src/main/java/com/blazebit/persistence/integration/jaxrs/EntityViewMessageBodyReader.java

```diff
@@ -67,6 +67,8 @@
     @Inject
     private Instance<EntityViewManager> entityViewManager;
     @Inject
+    private Instance<ObjectMapper> objectMapper;
```

Comment:
Normally in JAX-RS I use a `ContextResolver` for providing the `ObjectMapper`, as an `ObjectMapper` provided through a `ContextResolver` will also be recognized by the `ResteasyJackson2Provider`.


One would obtain this through:

```java
@Context
protected Providers _providers;
```

And then:

```java
ContextResolver<ObjectMapper> resolver = _providers.getContextResolver(ObjectMapper.class, mediaType);
```

Just my 2 cents that you could consider looking up an `ObjectMapper` though this as well (possibly extra), for applications that rely on this mechanism already.

Reply:
Thanks, I adapted that